src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -300,7 +299,6 @@ function WorkspaceTaxesPage({
                     ListItem={TableListItem}
                     customListHeader={getCustomListHeader()}
                     listHeaderContent={isSmallScreenWidth ? getHeaderText() : null}
-                    shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
```

Comment:
@tienifr IMO we should not remove the `shouldPreventDefaultFocusOnSelectRow`  prop from the SelectionList Component this will cause the unexpected result as this was added for some other reason

https://github.com/Expensify/App/blob/0c3b262e0cc21aee142e489f3c8aaa8d26189e1e/src/components/SelectionList/BaseSelectionList.tsx#L307-L309

https://github.com/Expensify/App/blob/0c3b262e0cc21aee142e489f3c8aaa8d26189e1e/src/components/SelectionList/BaseSelectionList.tsx#L315-L317

https://github.com/Expensify/App/blob/0c3b262e0cc21aee142e489f3c8aaa8d26189e1e/src/components/SelectionList/BaseSelectionList.tsx#L407